api/v3/routes/tracks.rb

```diff
@@ -32,10 +32,23 @@ class Tracks < Core
         }
       end
 
+      get '/tracks/:id/img/:filename' do |id, filename|
+        track = find_track(id)
+
+        img = track.img("img/#{filename}")
+        unless img.exists?
+          halt 404, {
+            error: "No image %s for track '%s'" % [filename, id],
+          }.to_json
+        end
```

Comment:
I wonder if we should default to the exercism logo if we don't have a track image. What do you think?